Documentation/Troubleshooting/ceph-common-issues.md

```diff
@@ -12,8 +12,10 @@ See also the [CSI Troubleshooting Guide](../Troubleshooting/ceph-csi-common-issu
 
 There are two main categories of information you will need to investigate issues in the cluster:
```

Comment:
I don't think we need to update this section. It's just a general statement about how to gather K8s and Ceph status. The specific areas like debug pods are details to be discussed in other docs.